example/lib/main.dart

```diff
@@ -1789,7 +1854,8 @@ class _MyAppState extends State<MyApp> {
       'date': CleverTapPlugin.getCleverTapDate(now),
       'number': 1
     };
-    CleverTapPlugin.recordEvent("Flutter Event", eventData);
+    //CleverTapPlugin.recordEvent("Flutter Event", eventData);
```

Comment:
Can we add a dynamic way of sending an event? Can ignore if it needs too many changes

Reply:
will do it as separate app enhancement.